src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -369,40 +433,73 @@ def getJobStatus(self, jobIDList):
             job, _, jobID = condorIDAndPathToResultFromJobRef(jobRef)
             condorIDs[job] = jobID
 
+        self.tokenFile = None
+
         qList = []
         for _condorIDs in breakListIntoChunks(condorIDs.values(), 100):
             # This will return a list of 1245.75 3
-            status, stdout_q = subprocess.getstatusoutput(
-                "condor_q {} {} -af:j JobStatus ".format(self.remoteScheddOptions, " ".join(_condorIDs))
-            )
+            cmd = ["condor_q"]
+            cmd.extend(self.remoteScheddOptions.strip().split(" "))
+            cmd.extend(_condorIDs)
+            cmd.extend(["-af:j", "JobStatus"])
+            result = self._executeCondorCommand(cmd, keepTokenFile=True)
+            if not result["OK"]:
+                self.tokenFile = None
+                return S_ERROR("condor_q failed completely: %s" % result["Message"])
```

Comment:
```suggestion
                return S_ERROR(f"condor_q failed completely: {result['Message']}")
```

Reply:
I guess this kind of changes is done in #6823 in general. This is the cause now of multiple conflicts